api/machines.go

```diff
@@ -59,7 +59,7 @@ func createTokenSecret() (*base64.Value, error) {
 }
 
 // List machines in the given org and state
-func (m *MachinesClient) List(ctx context.Context, orgIDs []*identity.ID, state *string, teamID *identity.ID) ([]*apitypes.MachineSegment, error) {
+func (m *MachinesClient) List(ctx context.Context, orgIDs []*identity.ID, state *string, names *[]string, teamIDs *[]*identity.ID) ([]*apitypes.MachineSegment, error) {
```

Comment:
these endpoints only support a single instance today, so we should probably match the behaviour